thread/thread.cpp

```diff
@@ -86,11 +86,15 @@ inline int posix_memalign(void** memptr, size_t alignment, size_t size) {
                            #name": "
 #endif
 
-static const size_t PAGE_SIZE = getpagesize();
-
 namespace photon
 {
     inline uint64_t min(uint64_t a, uint64_t b) { return (a<b) ? a : b; }
+
+    inline uint32_t get_pagesize() {
```

Comment:
How about put this inline function `get_pagesize()` into thread.h file,
so that same code could not appear twice in both `stack-allocator.cpp` and `thread.cpp`?